exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -45,11 +45,13 @@ drill.client: {
   supports-complex-types: true
 }
 
-// Directory is used as base for temporary storage of Dynamic UDF jars.
-// Set this property if you want to have custom temporary directory, instead of generated at runtime.
-// By default ${DRILL_TMP_DIR} is used if set.
-// drill.tmp-dir: "/tmp"
-// drill.tmp-dir: ${?DRILL_TMP_DIR}
+// Location Drill uses for temporary files, such as downloaded dynamic UDFs jars.
```

Comment:
1. Reverted generated directory logic.
2. "/tmp/drill/udf" + cluster-id is not good idea, since local udf directory has similar location as remote. And remote directory is generated as cluster-id + /udf. So I suggest to keep it as is. Local registry is returned to use its previous composition logic.
3. Removed "cluster-temp-dir", we don't need it for now.